File: Homework2/Homework2/Calculator.cs
namespace Homework1
{
    public static class Calculator
    {
        public static int Calculate(int value1, CalculatorOperation operation, int value2)
        {
            var result = operation switch
            {
                CalculatorOperation.Plus => value1 + value2,
                CalculatorOperation.Minus => value1 - value2,
                CalculatorOperation.Multiply => value1 * value2,
                CalculatorOperation.Divide => value1 / value2,
            };
            return result;
        }
    }
}

File: Homework4/Homework4/Calculator.fs
module Homework4.Calculator
let calculate value1 operation value2 =
    let value1d = value1 |> double
    let value2d = value2 |> double
    
    match operation with
    | CalculatorOperation.Plus -> value1d + value2d
    | CalculatorOperation.Minus -> value1d - value2d
    | CalculatorOperation.Divide -> value1d / value2d
    | _ -> value1d  * value2d